README.md

```diff
@@ -272,6 +272,13 @@ This is useful when installing your application on your device using the Develop
 
 You can statically select the right provisioning profile in your Xcode project (the name will be `match Development tools.fastlane.app`).
 
+### Continuous Integration
+
+When using match with a CI you'll need to do two things:
+
+- Set your encryption password as secret environment variable named `MATCH_PASSWORD`
+- Remember to give your CI access to the repo containing your keys, certs and profiles. You'll likely have to make a "bot" account on your Git host since you'll already have used the CI's deploy key for codebase repo.
```

Comment:
Yeah I agree, on third read it doesn't look so great. It's a tricky problem though, so I wanted to give others the heads up. It's not as easy as making another deploy key, because that's already been used for your codebase. Bitrise have [this](http://devcenter.bitrise.io/docs/adding-projects-with-submodules) article which gives a solution to the issue. Perhaps I should add a link?